.github/workflows/CI-CD-k8s-gear-js-stage.yml

```
@@ -58,6 +58,7 @@ jobs:
             VITE_NODES_API_URL= ${{ secrets.REACT_APP_DEFAULT_NODES_URL }}
             VITE_MAINNET_VOUCHERS_API_URL=${{ secrets.VITE_MAINNET_VOUCHERS_API_URL }}
             VITE_TESTNET_VOUCHERS_API_URL=${{ secrets.VITE_TESTNET_VOUCHERS_API_URL }}
+            VITE_DNS_API_URL=${{ secrets.VITE_TESTNET_DNS_API_URL }}
```

```suggestion
            VITE_DNS_API_URL=${{ secrets.VITE_DNS_API_URL }}
```

I guess there's a single url for both mainnet and testnet?